src/lib/renderComponent.tsx

```diff
@@ -159,6 +159,7 @@ const renderComponent = <P extends {}>(config: RenderConfig<P>): React.ReactElem
 
         const {
           siteVariables = {
+            colorScheme: {},
```

Comment:
not sure I follow your point @kuzhelov can you please expand or offer an example?

Reply:
@kuzhelov are you suggesting having maybe one `colors` object that will contain the scheme, colors  and everything related to the coloring mechanism? I would really avoid that refactoring as part of this part, but I can create an issue and open PR after this one gets in.

Reply:
@Bugaa92, sure - currently we have the following set of props being exposed by `siteVariables`: `colors`, `contextualColors`, `emphasisColors`, `colorScheme`, `naturalColors`, etc - there are just plenty of them, and all are aimed to describe `color` concept. Here I am proposing to introduce single `color` prop that will be an object with all these being assembled as props:

```js
siteVariables = {
   color: { 
      scheme, 
      contextual,
      natural,
       ...
   }
   ...
}
```

This will aid maintainability (as all closely related aspects will be semantically grouped), as well as will not pollute the `siteVariables` object itself. More than that, on the client perspective it will be much cleaner to see the entire set of objects from which colors could be consumed from, as everything related to `color` will be assembled under single object.

Reply:
@kuzhelov I like it, but would agree with @mnajdova to postpone this for a follow-up PR and create an issue for it